### DISCUSSION TITLE
Domain and dns

### DISCUSSION THREAD

**Discussion by franckbertuzzi** [facebook/docusaurus#8601] [General]
Hello

I would like to use docusaurus and netlify on my main domain > www. mysite .com

Problem : I have a subdomain used a php script, and the Dns are not those of netlify

What would be the solution ?

Thank you

**Comment by slorber** [comment]
> Problem : I have a subdomain used a php script

I don't understand what this means.

>  the Dns are not those of netlify

You don't need Netlify to control your DNS (ie setting Netlify nameservers) to be able to host Docusaurus on it. It's possible to configure your own DNS provider to target an existing deployment.

---

What's your DNS config. 

What's your existing subdomain.

Where do you want to host your Docusaurus site (final URL, including potential baseUrl)

What's your Netlify deployment config

Please be exhaustive if you want help
